kura/org.eclipse.kura.api/META-INF/MANIFEST.MF

```diff
@@ -29,13 +29,13 @@ Export-Package: org.eclipse.kura;version="1.3.0",
  org.eclipse.kura.gpio;version="1.0.1",
  org.eclipse.kura.linux.udev;version="1.0.0",
  org.eclipse.kura.message;version="1.1.0",
- org.eclipse.kura.net;version="1.2.1",
+ org.eclipse.kura.net;version="1.2.2",
```

Comment:
Actually, adding a method to this interface is a breaking change, so you should increase the major. And in any case, this should also be reflected in the Bundle-Version as well.

As more and more people are starting to use and extend Kura, it would be nice to try to keep API breakage to a minimum (arguably, if everyone is extending Kura by means of importing packages rather than requiring bundles, which is a good practice anyways, this will minimize the impacts for them only to those packages they really are depending on, but still…)

cc @dwoodard1 @cdealti @ctron